aws/aws-node-termination-handler
# upgrade test dependency versions

Issue #, if available:
N/A

Description of changes:
 - Upgrade helm to latest versions
 - Add k8s 1.19 kind docker image to e2e tests
 - patch version upgrade of k8s 1.18 kind docker image
 - Change default k8s version to 1.17
 - Remove k8s 1.12 and 1.13 since Kind no longer supports those version (latest supported eks is 1.14 so seems fine)
 - Enable rbac on AEMM

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
